cmd/kubelet/app/server.go

```diff
@@ -742,9 +742,9 @@ func parseResourceList(m kubeletconfiginternal.ConfigurationMap) (v1.ResourceLis
 			if q.Sign() == -1 {
 				return nil, fmt.Errorf("resource quantity for %q cannot be negative: %v", k, v)
 			}
-			// storage specified in configuration map is mapped to ResourceStorageScratch API
+			// storage specified in configuration map is mapped to ResourceEphemeralStorage API
 			if v1.ResourceName(k) == v1.ResourceStorage {
```

Comment:
I think we should change this too
if v1.ResourceName(k) == v1.ResourceEphemeralStorage {